src/main/webapp/i18n/en/exam.json

```diff
@@ -522,6 +522,8 @@
                     "shortNameTooltip": "The shortname of a programming task must be unique. The short name of the exercise can NOT be edited after creating the exercise! The short name must begin with a letter, must not contain any special characters and must be at least 3 characters long",
                     "invalidKey": "The chosen short name and title of {{ number }} programming exercise(s) already exists. Please insert a new short name and title for the corresponding programming exercise(s)",
                     "invalidExerciseConfiguration": "One or more of the values specified are invalid. Please change them before importing",
+                    "duplicatedProgrammingExerciseShortName": "Multiple programming exercises in the exam have the same short name. Please choose a different short name!",
+                    "duplicatedProgrammingExerciseTitle": "Multiple programming exercises in the exam have the same title. Please choose a different title!",
```

Comment:
In [src/main/webapp/i18n/de/exam.json](https://github.com/ls1intum/Artemis/pull/6877/files#diff-c782a9beef384a868bd33101e93f6e4d75301dc8dd025b7202704efd8163fb98) these lines where added directly beneath the entry for "invalidKey". It would be better to move those lines between "invalidKey" and "invalidExerciseConfiguration", too.

Reply:
The lines are now moved :+1: